python_scripts/trees.py

```diff
@@ -626,9 +632,10 @@ def plot_regression_model(X, y, model, extrapolate=False, ax=None):
 _ = plot_regression_model(X_train, y_train, tree)
 
 # %% [markdown]
-# We see that the decision tree model does not have a priori and do not end-up
+# We see that the decision tree model does not have a priori and we do not
+# end-up
 # with a straight line to regress flipper length and body mass. The prediction
-# of a new sample, which was already present in the training set, will give the
+# of a sample from the training set will give the
```

Comment:
I'm not sure exactly how to interpret the regression tree plot. How can one visualise the splits? Maybe a tree plot alongside would be helpful?

Also it's not obvious to me that predicting using a training sample will give the target of the training sample .. ?

Reply:
Below 'In the case of regression, the predicted value corresponds to the mean of the target in the node.' - I don't know how to tell what is a node in the graph.

Reply:
Below - 'extrapolate to unseen data' - do you mean extrapolate to unseen data outside of the range seen in the training dataset?